backend/src/main/java/gov/cdc/usds/simplereport/service/SpecimenService.java

```diff
@@ -241,4 +294,27 @@ private void saveSpecimens(List<Map<String, String>> rawSpecimens) {
     }
     // specimenRepository.saveAll(specimens);
   }
+
+  // TODO: Test saveSpecimenBodySites saves new bodySites
+  // TODO: Test saveSpecimenBodySites skips existing specimens  
+  private void saveSpecimenBodySites() {
+
+    if (!bodySites.isEmpty()) {
+      // TODO: We may want to move this duplicate check into the process
+      //  of populating the list of bodySites
+      for (SpecimenBodySite specimenBodySite : bodySites) {
+        SpecimenBodySite foundBodySite = specimenBodySiteRepository.findBySnomedSpecimenAndSiteCodes(
+                                                            specimenBodySite.snomedSpecimenCode,
+                                                            specimentBodySite.snomedSiteCode);
+        if (foundBodySite != null) {
```

Comment:
would we ever want to update an existing bodysite? or are these mostly static?

Reply:
And I realize I put the wrong code under this check condition, it should be continue.  Fixing.